osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -113,8 +117,21 @@ private void load(OsuConfigManager config, NotificationOverlay notificationOverl
             ModDisplay.Current.Value = mods;
 
             showHud = config.GetBindable<bool>(OsuSetting.ShowInterface);
-            showHud.ValueChanged += visible => visibilityContainer.FadeTo(visible.NewValue ? 1 : 0, duration);
-            showHud.TriggerChange();
+            showHud.BindValueChanged(visible => visibilityContainer.FadeTo(visible.NewValue ? 1 : 0, duration, easing), true);
+
+            ShowHealthbar.BindValueChanged(healthBar =>
+            {
+                if (healthBar.NewValue)
+                {
+                    HealthDisplay.FadeIn(duration, easing);
+                    topScoreContainer.MoveToY(30, duration, easing);
```

Comment:
@peppy why don't we use ternary operators in here
`HealthDisplay.FadeTo(healthBar.NewValue ? 1 : 0, duration, easing);`
`topScoreContainer.MoveToY(healthBar.NewValue ? 30 : 0, duration, easing);`
*unless you don't like it*

Reply:
i feel this way is more legible, personally.